src/components/MatchScore/MatchScore.tsx

```diff
@@ -48,7 +50,7 @@ export const MatchScore: FC<MatchScoreProps> = React.forwardRef(
 
                 {!hideLabel && (
                     <p className={matchScoreLabelClasses}>
-                        {absScore}/{absTotal}
+                        {label} {!hideValues && absScore + '/' + absTotal}
```

Comment:
is this scoring format really necessary? Can we just show the actual score? and not show it as a fraction? the visual gives a good indicator of the %

Reply:
Will open as a new pr, should be a choice between the two.